packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -71,45 +73,22 @@ export interface InternetGatewayProps {
 /**
  * Properties to define a VPN gateway.
  */
-export interface VPNGatewayProps {
-  /**
-   * The type of VPN connection the virtual private gateway supports.
-   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpngateway.html#cfn-ec2-vpngateway-type
-   */
-  readonly type: VpnConnectionType;
+export interface VPNGatewayV2Props extends VPNGatewayV2Options{
 
   /**
    * The ID of the VPC for which to create the VPN gateway.
    */
   readonly vpc: IVpcV2;
-
-  /**
-   * The private Autonomous System Number (ASN) for the Amazon side of a BGP session.
-   * @default none
-   */
-  readonly amazonSideAsn?: number;
-
-  /**
-   * The resource name of the VPN gateway.
-   * @default none
-   */
-  readonly vpnGatewayName?: string;
 }
 
 /**
- * Properties to define a NAT gateway.
+ * Options to define a NAT gateway.
  */
-export interface NatGatewayProps {
+export interface NatGatewayOptions {
   /**
    * The subnet in which the NAT gateway is located.
    */
-  readonly subnet: ISubnet;
```

Comment:
Isn't this a breaking change for users?

Reply:
if mean changing the ISubnet to ISubnetv2, its not breaking change since ISubnetv2 extends ISubnet